internal/fetch/jobDescription_test.go

```diff
@@ -1,13 +1,19 @@
 package fetch
 
 import (
+	"github.com/stretchr/testify/suite"
```

Comment:
could you please put it below stdlib imports (separated with an empty line), `goimports`-compatible?
```
import (
	"testing"

	"github.com/stretchr/testify/suite"
)
```

Reply:
Done